blueocean-rest/pom.xml

```diff
@@ -19,6 +19,10 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>blueocean-web</artifactId>
     </dependency>
+    <dependency>
```

Comment:
We don't want to depend on any jenkins or such plugin. Better to keep things abstract, if we can't its better to not expose such API from implementation.